CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -26,7 +26,7 @@
 using CursorType = CefSharp.Enums.CursorType;
 using Rect = CefSharp.Structs.Rect;
 
-namespace CefSharp.Wpf 
+namespace CefSharp.Wpf
```

Comment:
Are code changes actually required in this class? Looks like formatting only, this sort of thing makes reviews harder